Userland/Libraries/LibWeb/Streams/AbstractOperations.cpp

```diff
@@ -1626,6 +1730,43 @@ WebIDL::ExceptionOr<void> writable_stream_default_controller_process_write(Writa
     return {};
 }
 
+// https://streams.spec.whatwg.org/#writable-stream-default-controller-write
+WebIDL::ExceptionOr<void> writable_stream_default_controller_write(WritableStreamDefaultController& controller, JS::Value chunk, JS::Value chunk_size)
+{
+    auto& vm = controller.vm();
+
+    // 1. Let enqueueResult be EnqueueValueWithSize(controller, chunk, chunkSize).
+    auto enqueue_result = enqueue_value_with_size(controller, chunk, chunk_size);
```

Comment:
This doesn't build if the commit changing `enqueue_value_with_size` to take a value comes afterwards.